Dockerfile

```diff
@@ -1,8 +1,17 @@
-FROM golang:1.8.5-jessie
+FROM golang:alpine AS build
 # create a working directory
 WORKDIR /go/src/app
+#COPY go.mod go.sum ./
+#RUN go mod download
 # add source code
 COPY . .
 # run main.go
-RUN go run main.go
-#CMD ["bash"]
\ No newline at end of file
+ENV CGO_ENABLED=0
```

Comment:
Eu colocaria esse "ENV CGO_ENABLED=0" antes do "COPY . ." pra reduzir o número de camadas geradas cada vez que um código do projeto muda.
Mas de qualquer modo esse Dockerfile vai ser substituido pelo do veredito logo mais.

Reply:
Moved it up